frontend/src/pages/AppDetails/AppDetailsPage.tsx

```diff
@@ -279,7 +279,7 @@ class AppDetails extends React.Component<AppDetailsProps, AppDetailsState> {
 
 const mapStateToProps = (state: KialiAppState) => ({
   duration: durationSelector(state),
-  jaegerInfo: state.jaegerState.info,
+  jaegerInfo: state.tracingState.info,
```

Comment:
If the state is generic (tracingState), the variable should be renamed to tracingInfo.